public/app/plugins/datasource/cloudwatch/dataquery.gen.ts

```diff
@@ -10,6 +10,8 @@
 
 import * as common from '@grafana/schema';
 
+import * as common from '@grafana/schema';
```

Comment:
Issue: duplicated import.

This shows up when `common` is imported in `datasourcecfg.cue` like the following

```cue
import (
	common "github.com/grafana/grafana/packages/grafana-schema/src/common"
	"github.com/grafana/grafana/pkg/plugins/pfs"
)
```